dist/plugin/wavesurfer.elan.js

```diff
@@ -0,0 +1,271 @@
+(function (root, factory) {
+  if (typeof define === 'function' && define.amd) {
+    // AMD. Register as an anonymous module unless amdModuleId is set
+    define(["wavesurfer"], function (a0) {
+      return (factory(a0));
+    });
+  } else if (typeof exports === 'object') {
+    // Node. Does not work with strict CommonJS, but
+    // only CommonJS-like environments that support module.exports,
+    // like Node.
+    module.exports = factory(require("wavesurfer.js"));
+  } else {
+    factory(WaveSurfer);
+  }
+}(this, function (wavesurfer.js) {
```

Comment:
You can't call a variable `wavesurfer.js`

All the plugin builds have this issue